proposals/dao/fip_54.ts

```diff
@@ -52,10 +104,16 @@ export const teardown: TeardownUpgradeFunc = async (addresses, oldContracts, con
 };
 
 export const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts) => {
-  const { fei, core, restrictedPermissions } = contracts;
+  const { fei, tribe, core, restrictedPermissions, votiumBriberD3pool } = contracts;
```

Comment:
validate should check that the skimmers are all pointed to the correct PSM's

also, this pr needs to include the FeiSkimmer.sol file as it is not present

Reply:
manually confirmed after deploy